packages/hooks/package.json

```
@@ -5,6 +5,7 @@
     "main": "src/index.ts",
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.8",
+        "@deriv-com/analytics": "1.4.10",
```

could you use latest version? 
```suggestion
        "@deriv-com/analytics": "1.4.11",
```

---

sure